app/src/pod.js

```diff
@@ -132,6 +173,35 @@ export class Pod extends PIXI.Graphics {
         if (this.tick) {
             PIXI.ticker.shared.add(this.tick)
         }
+
+        const resources = this.getResourceUsage()
+
+        const cpuHeight = resources.cpu.limit !== 0 ? 8 / resources.cpu.limit : 0
+        podBox.lineStyle(0, 0xaaffaa, 1)
+        podBox.beginFill(getBarColor(resources.cpu.requested, resources.cpu.limit), 1)
+        podBox.drawRect(1, 9 - resources.cpu.requested * cpuHeight, 1, resources.cpu.requested * cpuHeight)
+        podBox.beginFill(getBarColor(resources.cpu.used, resources.cpu.limit), 1)
+        podBox.drawRect(2, 9 - resources.cpu.used * cpuHeight, 1, resources.cpu.used * cpuHeight)
+        podBox.endFill()
+        podBox.lineStyle(1, 0xaaaaaa, 1)
+
+        let limitExceeded = false
+        if (resources.memory.used > resources.memory.limit) {
+            resources.memory.limit = resources.memory.used
+            limitExceeded = true
```

Comment:
If this is done, most of the pods would blink because the API reports that most are using more than their limit.